index.js

```diff
@@ -0,0 +1 @@
+module.exports = require('./src/TimeSelect.jsx');
```

Comment:
Also its strange how this passes linting even though it has no `use strict` line.